### PR TITLE
New package: py-pyworld

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pyworld/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    (F0), aperiodicity and spectral envelope and also generate the speech like
+    input speech with only estimated parameters.i"""
+
+    homepage = "https://github.com/JeremyCCHsu/Python-Wrapper-for-World-Vocoder"
+    pypi     = "pyworld/pyworld-0.3.0.tar.gz"
+
+    version('0.3.0', sha256='e19b5d8445e0c4fc45ded71863aeaaf2680064b4626b0e7c90f72e9ace9f6b5b')
+
+    depends_on('py-setuptools',     type='build')
+    depends_on('py-numpy',          type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pyworld/package.py:21]: I wonder why the `pyproject.toml` says `py-numpy@:1.19`, did the package build with newer numpy? Does it run properly?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pyworld/package.py:21]: @qwertos

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-pyworld/package.py:21]: I'm not certain why. We have it successfully built against newer versions and haven't received any complaints from users.

ETA: I'm happy to add that version restriction if you keep consistent with the `pyproject.toml`

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pyworld/package.py:21]: Nah, if it works, it works

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**